docs/3.0-migration-guide.d/remove_mbedtls_check_params_option.md

```diff
@@ -0,0 +1,49 @@
+Remove MBEDTLS_CHECK_PARAMS option
+----------------------------------
+
+This change affects the way of how parameters are validated.
```

Comment:
I think here we want to start with "This change does not affect people who use the default configuration; in only affects users who enabled that option." - that way, a lot of users will immediately know they can skip the rest of that section.

Reply:
Done